doc/releases/release-notes-3.1.rst

```diff
@@ -768,6 +768,18 @@ Trusted Firmware-m
 Documentation
 *************
 
+* Reorganised and consolidated documentations for improved readability and
+  user experience.
+* Replaced the existing statically rendered Kconfig documentations with the new
```

Comment:
```suggestion
* Reorganised and consolidated documentation for improved readability and
  user experience.
* Replaced the existing statically rendered Kconfig documentation with the new
```

"Documentation" is almost always uncountable in my experience (wiktionary seems to agree: https://en.wiktionary.org/wiki/documentation, "very rare in the plural"). But not a blocking request.

Reply:
Agree, let´s fix this.

Reply:
fixed